modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6271,7 +6271,7 @@ fix-the-sla-configuration-to-resume-accurate-reporting=Fix the SLA configuration
 fixed=Fixed
 fixed-amount=Fixed Amount
 fixed-price=Fixed Price
-fixed-shipping-description=Ship for a fixed price.
+fixed-shipping-description=Flat Rate Description
```

Comment:
@garofalog are you sure about this fix? You are rewriting a label, but it is wrong.